### ISSUE TITLE
PHP Notice:  Undefined index: found_formatted in languages.php

### ISSUE THREAD

**Issue by slaFFik** [johnbillion/query-monitor#546]
Got this PHP notice:

```
PHP Notice:  Undefined index: found_formatted in ...\wp-content\plugins\query-monitor\output\html\languages.php on line 85
```

Unfortunately, I wasn't able to catch it earlier to provide steps. 
I was testing plugin loading its own translations properly when site language != user language inside WordPress.

**Comment by johnbillion** [comment]
Thanks for the report. I think this can happen when a plugin loads its translations too early, eg. before the `init` hook. I'll do some testing.

**Comment by ocean90** [comment]
I'm seeing the same notice in combination with Yoast SEO and the [Preferred Languages](https://wordpress.org/plugins/preferred-languages/) plugin:

<img width="1730" alt="Bildschirmfoto 2021-01-22 um 13 07 55" src="https://user-images.githubusercontent.com/617637/105489112-03f19f00-5cb3-11eb-9b60-d0c7f2d9c0e7.png">

The first two are not found, the last one is so `$mofile['found']` is true.

**Comment by johnbillion** [comment]
Thanks! That's a big help.

**Comment by glagonikas** [comment]
We too see quite a lot of those notices on our logs. Can't we just suppress them in the code by adding an `isset()`?

**Comment by johnbillion** [comment]
This should be fixed in 1.10, the property is no longer used.

**Comment by idejniprojekt** [comment]
I have same problem too: PHP Notice:  Undefined index: line in .../wp-content/plugins/query-monitor/output/html/languages.php on line 126

Any suggestion?
